tsconfig.json

```diff
@@ -5,13 +5,21 @@
     "declaration": true,
     "esModuleInterop": true,
     "isolatedModules": false,
-    "module": "CommonJS",
+    "module": "ESNext",
     "moduleResolution": "node",
     "noErrorTruncation": true,
     "noImplicitAny": true,
     "strict": true,
-    "target": "ESNext"
+    "target": "ESNext",
+    "resolveJsonModule": true,
   },
+  "files": [
+    // temporary for mock-api
+    "./packages/documents/src/pdf/al_name_change.json"
+  ],
+  "include": [
+    "./packages/*/src/**/*.json",
+  ],
```

Comment:
Various config changes to permit importing a local .json file in an idiomatic way